### DISCUSSION TITLE
dist-tag different from latest not showing in package version list

### DISCUSSION THREAD

**Discussion by alexandercerutti** [community/community#54247] [Packages]
### Select Topic Area

Product Feedback

### Body

Hi, we are publishing packages on Github NPM registry and we studied a versioning strategy to be based on NPM dist-tags.

We noticed that, on the website, only the `latest` dist-tag seems to get shown but not the others.

Since we were not seeing that on the UI after running `npm publish --tag <tag>`, we run `npm info <package>` and it shows that the dist-tag has been created. So it confirmed us that the dist-tag doesn't get shown in the UI.

Would it be possible to add it please?

Thank you very much.

**Comment by h2oearth** [comment]
Any update @github devs?

**Comment by jmaixl** [comment]
+1, we are experiencing this issue as well
